### PR TITLE
IA458 feat use color code for orgUnit statuses

### PR DESCRIPTION
**Pull request by quang-le**
## Changes

- Add color codes for Org Unit status: New = blue, validated = green, rejected=red
- green and red are the same colors as the `SnackBar`s
- blue is taken from `$primary-color` in  `_iaso.scss`

Screenshot 2021-07-05 at 13 44 56

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/orgUnits/utils.js [hunk 1]
```diff
@@ -174,6 +174,20 @@ export const getStatusMessage = (status, formatMessage) => {
     }
 };
 
+export const getStatusColor = status => {
+    switch (status) {
+        case 'NEW': {
+            // value taken from /iaso/hat/assets/css/_iaso.scss
+            return { color: '#2372BA' };
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:181]: this is `rawTheme.palette.primary.main` no ?

Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:181]: can we get it from the theme of the app ? with `useTheme` ?

Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:181]: I don't like the idea to hardcode the color

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/orgUnits/utils.js]: I made the change so we now use `theme.palette.primary.main`. It's purple though, but if you and @madewulf  think it fits the bill, it's all the same to me

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/orgUnits/utils.js]: It would be easier if I could access that value from the scss file in the js

**Pull request merged**